src/golang/cmd/server/handler/connect_integration.go

```diff
@@ -567,3 +571,21 @@ func validateLambdaConfig(
 
 	return http.StatusOK, nil
 }
+
+func validateConda(
```

Comment:
Since all we're doing is `conda --version`, I wonder if we have to go through the job manager to trigger this command. Unlike data integration validations, `conda --version` is a local operation that returns immediately, so it doesn't really make sense to fire the job and poll it. Also, we are already doing validations that bypass the job manager in our engine integrations above, so I don't think we have to restrict ourselves to using job manager here.